### DISCUSSION TITLE
addblocker - AWS Cloudfront

### DISCUSSION THREAD

**Discussion by MartijnHarmenzon** [plausible/analytics#2641] [Anything Else]
I am following these steps: https://plausible.io/docs/proxy/guides/cloudfront

### Question 1
When adding 
`<script defer data-domain="yourdomain.com" src="/js/script.js"></script>`
to the head of my document I will get this built error:

`GET http://127.0.0.1:3500/js/script.js net::ERR_ABORTED 404 (Not Found)`

Any suggestions on how to fix this?

### Question 2
Is the custom behaviour forthe event API required? 
`/api/event`

**Comment by metmarkosaric** [comment]
hi @MartijnHarmenzon! sorry for the trouble! seems like something went wrong with the first step of instructions as if that succeeded, you would not get an error message for the file itself? the event API is required to make the tracking work for the proxy

**Comment by MartijnHarmenzon** [comment]
Hi @metmarkosaric . Thanks for your reaction.
To clarify, this error only occurs when developping on localhost, where the file does not exist.
The build, deployed/hosted version works fine.

yourdomain.com/js/script.js results in status 200
yourdomain.com/api/event results in status 404

**Reply by metmarkosaric** [reply]
hmm i'm not sure why that happens on localhost. perhaps cloudfront docs or support might have an idea? seems related to their platform rather than something we can do
